app/controllers/invitations_global_controller.rb

```diff
@@ -0,0 +1,45 @@
+class InvitationsGlobalController < ApplicationController
```

Comment:
@shir I think it is to better name models `Invitation`, `OrganizationInvitation` and `InvitationBase`, what do you think?